### PR TITLE
Adding IDMind drivers

### PR DESCRIPTION
**Pull request by duartegameiro**
Adding IDMind drivers to control LEDs, Neck, Head and Camera Tilt motors.

### CODE REVIEW

**Comment by ipa-nhg** [comment]
Hi @duartegameiro ,

i sent you a pull request with my suggestions https://github.com/duartegameiro/squirrel_driver/pull/1, what I did:

- move all the ros packages to the same level (under squirrel_driver) and not under a new folder idmind_drivers/packageName
- harmonize of the package.xml: version number and format
- give executable rights to the python files
- clean up the CMakeLists

Please comment the pull request if you don't understand something or you prefer to keep your version.

Thanks!

File: idmind_interaction/package.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package>
+  <name>idmind_interaction</name>
+  <version>0.0.1</version>
+  <description>The idmind_interaction package</description>
+  <maintainer email="dgameiro@idmind.pt">Duarte Lopes Gameiro</maintainer>
+  <license>TODO</license>
```
Comment (NadiaHG) [inline idmind_interaction/package.xml]: please add license type

File: idmind_teleop/package.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<package>
+  <name>idmind_teleop</name>
+  <version>0.0.1</version>
+  <description>The idmind_teleop package</description>
+  <maintainer email="dgameiro@idmind.pt">Duarte Gameiro</maintainer>
+  <license>TODO</license>
```
Comment (NadiaHG) [inline idmind_teleop/package.xml]: please add license type

File: raspberry_screen/package.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<package>
+  <name>raspberry_screen</name>
+  <version>0.0.1</version>
+  <description>Raspberry screen service</description>
+  <maintainer email="mcorberan@idmind.pt">Miguel Corberán Ruiz</maintainer>
+  <license>TODO</license>
```
Comment (NadiaHG) [inline raspberry_screen/package.xml]: please add license type

**Review by ipa-nhg — COMMENTED** [review]
@duartegameiro just a small comment. could you please add the license types?

**Comment by duartegameiro** [comment]
I already updated the files.
Thanks!

**Review by ipa-nhg — APPROVED** [review]

**Pull request merged**